asgi_ipc.py

```diff
@@ -42,15 +42,11 @@ def __init__(self, prefix="asgi", expiry=60, group_expiry=86400,
         )
         self.thread_lock = threading.Lock()
         self.prefix = prefix
-        self.channel_store = MemoryDict(
-            "/%s-chan" % self.prefix,
-            size=channel_memory
-        )
+        connection = sqlite3.connect(tempfile.NamedTemporaryFile().name)
```

Comment:
I have now, of course, realised that the temporary file name has to be predictable, so different channel layer instances in different processes agree on it, so your previous way was actually right. You can still use `tempfile.gettempdir()` rather than hardcoding `/tmp` though.